google-cloud-pubsub/src/test/java/com/google/cloud/pubsub/it/ITPubSubTest.java

```diff
@@ -89,23 +93,46 @@ private String formatForTest(String resourceName) {
     return resourceName + "-" + NAME_SUFFIX;
   }
 
+  private Subscription getSubscription(
+      String subscriptionName, String topicName, PushConfig pushConfig, int ackDeadline) {
```

Comment:
Can we take in SubscriptionName and TopicName here, and do the toString conversions within this method?